sagafalabella/sagafalabella/spiders/falabella.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from urllib.parse import quote_plus
+import json
+from scrapy import Request
+
+class FalabellaSpider(scrapy.Spider):
+    name = 'falabella'
+    allowed_domains = ['falabella.com.pe/falabella-pe']
+    start_urls = ['http://falabella.com.pe/falabella-pe']
+
+    def parse(self, response):
+        # we are not interested in some anchors give us url to all categories
+        categories = response.xpath(
+            '//ul[@class="fb-masthead__grandchild-links"]/li/a[not(contains(text(),"Ver todo"))]/@href').extract()
+        for categorie in categories:
+            categorie_url = response.urljoin(categorie)
+            yield Request(categorie_url, callback=self.parse_categorie, dont_filter=True)
```

Comment:
You could use `response.follow`: https://doc.scrapy.org/en/latest/topics/request-response.html#scrapy.http.Response.follow